src/app/components/info-drawer/info.drawer.module.ts

```diff
@@ -61,10 +60,11 @@ export function components() {
     ContentMetadataModule,
     VersionManagerModule,
     DirectivesModule,
-    A11yModule
+    A11yModule,
+    SharedInfoDrawerModule
   ],
   declarations: [...components()],
-  exports: [...components()],
+  exports: [...components(), SharedInfoDrawerModule],
```

Comment:
not needed as it's already provided by the "components()" function

Reply:
This is required, cause we moved the few components to the shared library. 
so if we want to use a shared library component while importing the `appToolbarModul`e, we need to export.

Reply:
@dhrn  the `...components()` code already exports it. You are exporting the module 2 times

Reply:
I guess you are missing here
I just removed the components (reusable) there. I will not export two times

Reply:
ok, I see, thanks

Reply:
I guess you are missing here
I removed the reusable components from `components()` this function. so I hope it will now export two times

am i wrong?